### DISCUSSION TITLE
An .exe file can not be generated automatically for .cpp file, but it did work three month ago.

### DISCUSSION THREAD

**Discussion by YamakiYume** [microsoft/vscode-cpptools#10302] [Q&A]
I wrote some C++ codes before, and they ran successfully at that time with MingW and VScode. We mainly used C this semester, so I haven't run C++ codes for a long time. I tried to run them today, but a '...\....exe does not exist' was on my screen. I searched on the internet, checked g++ gdb and .json files(They are all OK), even made an .exe file manually. However, .exe file disappeared and '...\....exe does not exist' again when I ran the code.
How can I solve this problem?

**Comment by sean-mcmanus** [comment]
We need more info. How are you building? With tasks.json? What task type? Are you saying the exe is generated and then something is deleting it? What version of the C/C++ extension are you using?

**Reply by YamakiYume** [reply]
I am sorry that I didn't notice the message. I have no idea why it worked last month itself, but thanks for your help!
